File: Iniciante/1038 - Lanche/TypeScript/1038.ts
import * as fs from "fs";

const PRICE_TABLE: number[] = [4.00, 4.50, 5.00, 2.00, 1.50];

function main(): void {
    const input: string = fs.readFileSync("/dev/stdin", "utf8");
    const lines: string[] = input.trim().split(/\s+/);
    const [code, amount] = lines.map(Number);
    const totalBill: number = calculateTotalBill(code, amount);
    console.log(`Total: R$ ${totalBill.toFixed(2)}`);
}

function calculateTotalBill(code: number, amount: number): number {
    return PRICE_TABLE[code - 1] * amount;
}

main();

File: Iniciante/1038 - Lanche/Python/1038.py
SNACK_MENU = {1: 4.00, 2: 4.50, 3: 5.00, 4: 2.00, 5: 1.50}

def main():
    code, amount = map(int, input().split())
    total_bill = calculate_total_bill(code, amount)
    print(f"Total: R$ {total_bill:.2f}")


def calculate_total_bill(code, amount):
    return SNACK_MENU[code] * amount


if __name__ == "__main__":
    main()